packages/contracts-bedrock/test/properties/medusa/handlers/Protocol.t.sol

```diff
@@ -159,16 +159,20 @@ contract ProtocolHandler is TestBase, StdUtils, Actors {
         // what we use in the tests to walk around two contracts needing two different addresses
         // tbf we could be using CREATE1, but this feels more verbose
```

Comment:
and would have avoided the pain of c2 collision 🤦

Reply:
we'd have the alternate pain of deployments for what would be the same supertoken (as in, one unique realSalt +chainId) silently overwriting previous ones :D and potentially breaking invariants later, instead of failing eagerly